### DISCUSSION TITLE
tutorials/graphos-shipping-supergraph/02-project-setup

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#118] [course feedback]
# tutorials/graphos-shipping-supergraph/02-project-setup

https://www.apollographql.com/tutorials/graphos-shipping-supergraph/02-project-setup

**Comment by JTMonaghan** [comment]
On step 2 of the Railway deployment steps, instead of being able to select a cloning destination within my linked Github acct, all I get is an uneditable link to the poetic-plates-recipes-api repo on Github. If I try clicking the Deploy button, nothing happens. Any thoughts?

**Reply by mabuyo** [reply]
Hmm, could you post a screenshot here of what that looks like? It's also possible Railway has changed something recently. What plan are you on?

**Reply by JTMonaghan** [reply]
I made the Railway account to go through the tutorial, so I just have the Trial Plan. I might be missing something, but I don't see a way to post a screenshot here?

**Reply by mabuyo** [reply]
Oh sorry about that - [try posting the screenshot here](https://github.com/apollographql-education/content-feedback/discussions/118#discussioncomment-9194021).

**Reply by JTMonaghan** [reply]
Ah, thanks! I was posting from the Tutorial page. 
![image](https://github.com/apollographql-education/content-feedback/assets/10657744/e979d176-9535-4c7a-b245-4feb172308b0)

**Reply by mabuyo** [reply]
Thank you! It looks like I'm getting the same thing, so something has changed on their end... I'll take a look into this. For now, you can unblock yourself by:

1. Fork the [repo](https://github.com/apollographql-education/poetic-plates-recipes-api) on GitHub so you have your own copy.
2. Create a new project on Railway, selecting "Deploy from GitHub repo", then select your repo.

Then you can continue on with the lesson. Let me know if you run into any issues!

**Reply by JTMonaghan** [reply]
Worked great, thank you @mabuyo!
